packages/crawler/tests/e2e/config/jest.config.ts

```diff
@@ -0,0 +1,31 @@
+const defaultWorkers = process.env.CI === 'true' ? 1 : 2;
```

Comment:
Why duplicate instead of reusing the shared one?

Reply:
Fixed it - now we use shared config